kwok/inittab

```diff
@@ -1,2 +1,4 @@
-::wait:/usr/local/bin/kwokctl start cluster --name kwok
+::wait:/start-clusters.sh
```

Comment:
AFAIK you can have multiple "wait" statements, so no need of a script.

Reply:
I started with that, and ran into some issues, but in hindsight they might have been user issues. I'll give it another try.

Reply:
There seems to be no issue with multiple wait statements.
However, there seems to be an issue for me when starting a cluster after another - it seems to fail often.

The host is started in a second or so, then the first member is hanging until the timeout is reached and then the second member is starting in a second or so as well.

Reply:
so did using a script solve that somehow?

Reply:
That's wasn't my original issue. The script did not have that issue, but I think the script is not the way to go. I think adding a short sleep before starting the next cluster addresses the issue, but I'm looking into that.

Reply:
I was not able to figure it out yet. It's working --- eventually. I'll try to explore a bit further. It might not happen today.

Reply:
We can introduce a script that will run the same startup command but will allow a few retries in case it fails. What do you think?

Reply:
I had a look at the set of running processes for a cluster:
```
  277 root      0:00 /root/.kwok/clusters/kwok/bin/etcd --name=node0 --auto-compaction-retention=1 --quota-backend-bytes=8589934592 --data-dir=/root/.kwok/clusters/kwok/etcd --initial-advertise-peer-urls=http://0.0.0.0:32766 --listen-pee
  278 root      0:02 /root/.kwok/clusters/kwok/bin/kube-apiserver --etcd-prefix=/registry --allow-privileged=true --etcd-servers=http://127.0.0.1:32765 --authorization-mode=Node,RBAC --bind-address=0.0.0.0 --secure-port=8080 --tls-cert-f
  279 root      0:00 /root/.kwok/clusters/kwok/bin/kwok-controller --manage-all-nodes=true --kubeconfig=/root/.kwok/clusters/kwok/kubeconfig.yaml --config=/root/.kwok/clusters/kwok/kwok.yaml --tls-cert-file=/root/.kwok/clusters/kwok/pki/
  280 root      0:00 /root/.kwok/clusters/kwok/bin/kube-controller-manager --node-monitor-period=10m0s --node-monitor-grace-period=1h0m0s --kubeconfig=/root/.kwok/clusters/kwok/kubeconfig.yaml --authorization-always-allow-paths=/healthz,
  281 root      0:00 /root/.kwok/clusters/kwok/bin/kube-scheduler --kubeconfig=/root/.kwok/clusters/kwok/kubeconfig.yaml --authorization-always-allow-paths=/healthz,/readyz,/livez,/metrics --bind-address=0.0.0.0 --secure-port=32762
```

This seems heavy, and it makes me think we have an issue with the resources in the container. I suspect we don't need all of that. 

Perhaps try disabling what you can in the `cluster create` command, we probably only need etcd and the API server.

Reply:
I'm able to remove `kube-controller-manager` and `kube-scheduler`, but it does not seem to affect the ability to get the container up. Yesterday, before the end of the day I could not make it fail, and this morning I had to run `podman system reset` to be able to make it start.

Also, waiting few seconds between starting the clusters doesn't make a difference. 

It's something about resources, but I'm not able to tell what it is.